src/l10n/nb.ts

```diff
@@ -6,7 +6,7 @@ export default {
   no: 'Nei',
   protocolApplyTo: 'Denne behandlingen gjelder',
   records: 'Behandlingsaktiviteter',
-  dataProcessor: 'Behandlingsansvarlig og databehandler',
+  dataProcessor: 'Daglig behandlingsansvar og databehandler',
```

Comment:
Ser ut til at jeg sa feil. Når jeg ser nærmere på issuet så ser det ut til at teksten du skal endre ligger i `components/record-form/index.tsx`

https://github.com/Informasjonsforvaltning/records-of-processing-activities-gui/blob/main/src/components/record-form/index.tsx#L247

Reply:
No problem. Skal fikse det nå.